v3/fleet/control-proxy.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=run apigateway as control proxy
+Requires=docker.service
+After=docker.service bootstrap.service zk-health.service
+
+[Service]
+Environment="IMAGE=etcdctl get /images/control-proxy"
+EnvironmentFile=/etc/environment
+User=core
+Type=simple
+Restart=always
+RestartSec=4
+TimeoutStartSec=0
+ExecStartPre=/usr/bin/sh -c "docker pull $($IMAGE)"
+ExecStartPre=-/usr/bin/docker kill control-proxy
+ExecStartPre=-/usr/bin/docker rm control-proxy
+ExecStart=/usr/bin/bash -c \
+  '/usr/bin/docker run \
+  --name control-proxy \
+  -e MESOS_MASTER_HOST=http://`etcdctl get /environment/CONTROL_ELB`:5050 \
```

Comment:
we can pass this as variable: `etcdctl get /environment/CONTROL_ELB` in environment